archlinuxcn/mesa-git/PKGBUILD

```diff
@@ -91,7 +91,6 @@ package_vulkan-mesa-layers-git() {
   provides=('vulkan-mesa-layers')
   depends=('wayland' 'libxcb' 'python')
   conflicts=('vulkan-mesa-layer-git' 'vulkan-mesa-layers')
-  replaces=('vulkan-mesa-layer-git')
```

Comment:
Assuming he package used to be called vulkan-mesa-layer (without s) this should stay as is.